Docs 9549 - Updates for editorial review (#106)
mulesoft/docs-partner-manager: modules/ROOT/pages/outbound-message-routing.adoc

= Outbound Message Routing

> Partner Manager’s outbound message processing framework provides the flexibility to implement receive endpoints in a variety of ways.

Reviewer: Avoid making names possessive:
```suggestion
The Partner Manager outbound message processing framework enables you to implement receive endpoints in a variety of ways:
```
Reword list to work with this sentence as its intro., instead of using "You can:"

> ** Relative path for HTTP(s) endpoints

Reviewer: These look like they should be numbered actions rather than bullets.
Also, GLOBALLY replace "HTTP(s)" or "HTTP(S)" by naming both: HTTP or HTTPS.

Reviewer: Fix capitalization typo of "identifies" in line 13.

Author: Done - will do global check in a different PR

** File name for SFTP

> * The payload is validated against the schema supplied during the message type definition.

Reviewer: passive

Author: Fixed

> * Partner Manager leverages the DataWeave map supplied during the source at host message type definition to extract the partner reference ID, host reference ID, and business key.

Reviewer: Can we reword this...noun?
"the DataWeave map supplied during the source at host message type definition"
Also, "leverages to extract" is wordy and vague.
Why not something simple like "Partner Manager uses the supplied DataWeave map to obtain the partner and host reference IDs and the business key."

Author: Done

> * The partner reference identifier and host reference identifier (optional) are used to resolve the message flow through which the transaction must be processed as shown in the following diagram:

Reviewer: passive
```suggestion
* The partner reference identifier and optional host reference identifier specify the message flow through which the transaction is processed:
```
